frontend-react/src/App.tsx

```diff
@@ -27,6 +27,8 @@ import { IS_IE } from "./utils/GetIsIE";
 
 const { APP_ENV } = config;
 
+window.alert("HEY THIS IS COOL");
```

Comment:
Will remove before merge, but I wanted to make sure this was actually working on the trialfrontend01 environment :)